letsencrypt-auto-source/pieces/bootstrappers/deb_common.sh

```diff
@@ -18,11 +20,18 @@ BootstrapDebCommon() {
   # - Debian 6.0.10 "squeeze" (x64)
 
   $SUDO apt-get update || echo apt-get update hit problems but continuing anyway...
+  
+  # checking if certbot/letsencrypt is already packaged with OS
+  if [ "$QUIET" != 1 ]; then
+      if dpkg --get-selections | grep -v letsencrypt.sh | grep -c letsencrypt -gt 0; then
+        echo "Certbot is already packaged with your OS."
+        dpkg --get-selections | grep -v letsencrypt.sh | grep letsencrypt
+      fi
+  fi
 
   # virtualenv binary can be found in different packages depending on
   # distro version (#346)
-
-  virtualenv=
+    virtualenv=
```

Comment:
Yes it was unintentional. I have reverted this change. I would check the diff properly before creating a pull request from now on.